offchain/danaswap-browser/src/DanaSwap/Browser/Home.purs

```diff
@@ -16,21 +22,248 @@ main = HA.runHalogenAff do
+              [ HH.div [ mkClass "field" ]
+                  [ HH.div [ mkClass "control" ] [ HH.input [ HP.id "swap-asset-a", mkClass "input is-rounded" ] ]
+                  ]
+              ]
+          , HH.div [ mkClass "column is-1" ]
+              [ HH.label [ HP.for "swap-asset-a", mkClass "label is-normal" ] [ HH.text a.name ]
+              ]
+          ]
+      , HH.div [ mkClass "columns is-mobile is-vcentered" ]
+          [ HH.div [ mkClass "column is-offset-1-desktop is-offset-3-touch is-1 has-text-centered" ]
+              [ HH.span
+                  [ mkClass "icon is-medium" ]
+                  [ HH.i [ mkClass $ "fas fa-exchange-alt" ] [] ]
+              ]
+          ]
+      , HH.div [ mkClass "columns is-mobile is-vcentered" ]
+          [ HH.div [ mkClass "column is-8-touch is-3-desktop" ]
+              [ HH.div [ mkClass "field" ]
+                  [ HH.div [ mkClass "control" ] [ HH.input [ HP.id "swap-asset-b", mkClass "input is-rounded" ] ]
+                  ]
+              ]
+          , HH.div [ mkClass "column is-1" ]
+              [ HH.label [ HP.for "swap-asset-b", mkClass "label is-normal" ] [ HH.text b.name ]
+              ]
+          ]
+      , HH.p [ mkClass "subtitle is-6 my-0" ]
+          [ HH.text "Slippage: 4%"
+          ]
+      , HH.p [ mkClass "subtitle is-6" ]
+          [ HH.text "Fee: 2.3 ADA"
+          ]
+      , HH.div [ mkClass "columns" ]
+          [ HH.div [ mkClass "column is-narrow" ]
+              [ HH.button [ HP.type_ ButtonSubmit, mkClass $ "button is-rounded is-medium is-responsive danaswap-btn-has-border" ] [ HH.text "submit" ]
+              ]
+          ]
       ]
+
+    renderAddLiquidityForm (Pool ({ assetA: (Asset a), assetB: (Asset b) })) = HH.form [ mkClass "mt-6" ]
+      [ HH.div [ mkClass "columns is-mobile is-vcentered" ]
+          [ HH.div [ mkClass "column is-8-touch is-3-desktop" ]
+              [ HH.div [ mkClass "field" ]
+                  [ HH.div [ mkClass "control" ] [ HH.input [ HP.id "add-liquidity-asset-a", mkClass "input is-rounded" ] ]
+                  ]
+              ]
+          , HH.div [ mkClass "column is-1" ]
+              [ HH.label [ HP.for "add-liquidity-asset-a", mkClass "label is-normal" ] [ HH.text a.name ]
+              ]
+          ]
+      , HH.div [ mkClass "columns is-mobile is-vcentered" ]
+          [ HH.div [ mkClass "column is-8-touch is-3-desktop" ]
+              [ HH.div [ mkClass "field" ]
+                  [ HH.div [ mkClass "control" ] [ HH.input [ HP.id "add-liquidity-asset-b", mkClass "input is-rounded" ] ]
+                  ]
+              ]
+          , HH.div [ mkClass "column is-1" ]
+              [ HH.label [ HP.for "add-liquidity-asset-b", mkClass "label is-normal" ] [ HH.text b.name ]
+              ]
+          ]
+      , HH.p [ mkClass "subtitle is-6 my-0" ]
+          [ HH.text "Liquidity Tokens: 0"
+          ]
+      , HH.p [ mkClass "subtitle is-6" ]
```

Comment:
Curious - is there tooling that can compile the stylesheet into a module so we can reference the styles through a module namespace rather than use hard-coded strings?

Reply:
@EvanPiro there's a binding lib for bulma (the CSS framework we're using for Danaswap), but it was not maintained any more, last time it was updated is 4 years ago https://github.com/sectore/purescript-bulma